WalletWasabi.Fluent/ViewModels/Wallets/Receive/ReceiveAddressViewModel.cs

```diff
@@ -100,7 +100,7 @@ private void GenerateQrCode()
 		{
 			try
 			{
-				QrCode = new QrEncoder().Encode(Address).Matrix.InternalArray;
+				QrCode = new QrEncoder().Encode(Address.ToUpper()).Matrix.InternalArray;
```

Comment:
Consider using ToUpperInvariant. You never know how casing works for some languages.

Reply:
the output of bech32 encoding should be ascii only, would that still be relevant? at any rate based on the docs this suggestion makes sense to me

Reply:
FTR: bech32 has a human-readable part that allow ASCII from [33-126] and that part can be problematic for some cultures. for example:
```c#
"stillhere".ToUpper(new CultureInfo("tr-TR")); // STİLLHERE  instead of STILLHERE 
```